tests/test_doctests.py

```diff
@@ -5,15 +5,20 @@
 Run selected doctests as a regular unit test.
 """
 
-import os
+import doctest
 import sys
+import unittest
+
```

Comment:
one newline too many here

Reply:
PyCharm put that one there. I'm not sure why os is different from sys.

Will fix.